src/src/pages/capabilities/SelectedCapabilityContext.js

```diff
@@ -358,6 +375,17 @@ function SelectedCapabilityProvider({ children }) {
     }
   }, [isLoadedMembers, membersList]);
 
+  useEffect(() => {
+    if (details) {
+      loadMembershipApplications();
+      getBypassMembershipApplication();
```

Comment:
bad naming: you are not only getting, you are first fetching and then setting a variable